go.mod

```diff
@@ -34,3 +34,5 @@ require (
 	gopkg.in/yaml.v2 v2.2.1 // indirect
 	qiniupkg.com/x v7.0.7+incompatible // indirect
 )
+
+go 1.13
```

Comment:
go 1.12?

Reply:
it's automatically generated by `go mod init`... I can delete this